web/docs/auth/social-auth/_getuserfields-type.md

```diff
@@ -1,3 +1,3 @@
-Wasp automatically generates the type `GetUserFieldsFn` to help you correctly type your `getUserFields` function.
+Wasp automatically generates the `defineUserFields` function to help you correctly type your `getUserFields` function.
```

Comment:
Aha, that is what it does! Ok, so it types them. Anything else it does? How is it that you switched from type to "hook"?

Reply:
This is something I worked on when we aded the `additionalSignupFields` feature. It's hard to type the object and force some rules regarding to generics without using an [identity function](https://www.typescriptlang.org/docs/handbook/2/generics.html) like this.